### PR TITLE
Fabio's (lh536) image added

### PR DESCRIPTION
**Pull request by lh536**
## Description
I added my image to TEAM.md

## Attach Screenshot
image

> Note 2 code reviewer approval needed. Approach in twitter group & discord channel.

### CODE REVIEW

**Review by pranavgoel29 — APPROVED** [review]
Good to go.

**Review by shekhar10feb — APPROVED** [review]

**Comment by lh536** [comment]
@proful Can you review my PR? Thank you!

**Comment by lh536** [comment]
> Good to go.

Thank you for your support!!!

**Comment by pranavgoel29** [comment]
> > Good to go.
> 
> Thank you for your support!!!

No problem 👍🏻🥳

File: docs/TEAM.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Beginner
 
+
```
Comment (ni3-kdm) [inline docs/TEAM.md:3]: @lh536 Remove these new lines. Doesn't look needful.

File: docs/TEAM.md [hunk 2]
```diff
@@ -38,6 +40,7 @@
 |                                                                                          | [Lorenna Leon](https://github.com/lorennaleon)               | I am Peruvian, I have knowledge in html, css and javascripts I hope to learn a lot from you                                                                                                                                                                                                                                                                                                                |
 | <img src="https://avatars2.githubusercontent.com/u/58055517" width="90px" height="90px"> | [Fusen Ye](https://github.com/Yexiansen66)                   | I'm a beginner from China. I know HTML,CSS and JavaScript.I want to learn more advanced knowledge and contribute to twindle.                                                                                                                                                                                                                                                                               |
 
+
```
Comment (ni3-kdm) [inline docs/TEAM.md:43]: @lh536 Remove these new lines. Doesn't look needful.

**Review by praveen2896 — APPROVED** [review]
Looks good .

**Pull request merged**